modules/core/src/v2/coins/cspr.ts

```diff
@@ -58,6 +58,13 @@ interface TransactionOutput {
   coin: string;
 }
 
+interface TransactionOperation {
+  type: string;
+  amount: string;
+  coin: string;
+  validator?: string;
```

Comment:
Why validator is optional?

Reply:
Maybe some types of operations not include it... Like Wallet initialization.